movement/sample_data.py

```diff
@@ -295,7 +295,7 @@ def fetch_dataset(
         if file_paths.get(key):
             ds = load_module.from_file(
                 file_paths[key],
-                source_software=metadata[filename]["source_software"],
+                source_format=metadata[filename]["source_format"],
```

Comment:
This is to make the `test_sample_data` tests pass, until we change the name of this field to "source_format" also on our data repository.
```suggestion
                source_format=metadata[filename]["source_software"],
```